playback/src/player.rs

```diff
@@ -1305,14 +1305,20 @@ impl PlayerInternal {
                                 // Engineering Society, 60, 399-408.
                                 if self.config.normalisation_method == NormalisationMethod::Dynamic
                                 {
-                                    // steps 1 + 2: half-wave rectification and conversion into dB
-                                    let abs_sample_db = ratio_to_db(sample.abs());
+                                    if !(self.normalisation_integrator > 0.0
+                                        || self.normalisation_peak > 0.0)
```

Comment:
```rust

                                    if !(limiter_input > 0.0
                                        || self.normalisation_integrator > 0.0
                                        || self.normalisation_peak > 0.0)
                                    {
```

I was basing everything off your code. This later in your code would have the same effect. Because up until that point all you've done is a bunch of math that doesn't the effect the instance vars.

Reply:
No, that also proceeds if `limiter_input > 0.0`. You propose to *not* proceed without checking `limiter_input`. Again this causes the entire limiter to stop working. Try adding a trace message after, and you'll see.

Reply:
You said:

> This is incorrect because limiter_input is always >= 0.0

So that part is useless. It's always a noop.

Reply:
No, it skips when `limiter_input` is `0.0`.